mlir/test/Dialect/Linalg/fold-add-into-dest.mlir

```diff
@@ -0,0 +1,288 @@
+// RUN: mlir-opt %s -transform-interpreter -cse -split-input-file | FileCheck %s
+
+!type = tensor<2048x2048xf32>
+func.func @expect_add_to_fold(%arg0: !type, %arg1: !type) -> !type {
```

Comment:
[nit] Could we have more descriptive function names? In particular, what's the difference between this version of `@expect_add_to_fold` and the one below?

Reply:
Fair - have changed the names to reflect the differences in test cases.